### PR TITLE
Copy up release channel and CRDs

### PR DESCRIPTION
**Pull request by TerryHowe**
This artifacts need to be pushed for 1-19

### CODE REVIEW

File: release/crds/releasechannels.distro.eks.amazonaws.com-v1alpha1.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
```
Comment (vivek-koppuru) [inline release/crds/releasechannels.distro.eks.amazonaws.com-v1alpha1.yaml:4]: Shouldn't CRDs be defined in our build-tooling repo like here or are we moving them here instead? https://github.com/aws/eks-distro-build-tooling/tree/main/release/config/crds

File: release/1-18/1-18.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+apiVersion: distro.eks.amazonaws.com/v1alpha1
```
Comment (vivek-koppuru) [inline release/1-18/1-18.yaml:1]: Let's add the Apache License configuration to these files as well

Comment (TerryHowe) [reply release/1-18/1-18.yaml:1]: While # is a valid comment in yaml, I'm not a huge fan of uploading these artifacts with comments in them, nor am I a big fan of some logic to strip the comments off.

File: release/crds/releasechannels.distro.eks.amazonaws.com-v1alpha1.yaml (hunk 1 above)
Comment (TerryHowe) [reply release/crds/releasechannels.distro.eks.amazonaws.com-v1alpha1.yaml:4]: Having two repos is a constant source of extra work and the risk of more moving parts, which is why I put them here.

File: release/1-18/1-18.yaml (hunk 2 above)
Comment (vivek-koppuru) [reply release/1-18/1-18.yaml:1]: Why do we need to strip comments? Also it's a requirement to add the Apache License to our source code I thought.

Comment (TerryHowe) [reply release/1-18/1-18.yaml:1]: It just seems really odd to have that copyright comment in the middle of an artifact. We can just skip automating this and someone can copy these files manually with no copyright.

File: release/crds/releasechannels.distro.eks.amazonaws.com-v1alpha1.yaml (hunk 1 above)
Comment (TerryHowe) [reply release/crds/releasechannels.distro.eks.amazonaws.com-v1alpha1.yaml]: Okay, I changed this to pull the CRDs out of the other repo

File: release/1-18/1-18.yaml (hunk 2 above)
Comment (TerryHowe) [reply release/1-18/1-18.yaml]: I changed this to grep out the comments before uploading them.

**Review by kschumy — APPROVED** [review]
/lgtm

**Pull request merged**